.eslintrc.yaml

```diff
@@ -16,32 +18,15 @@ extends:
   - plugin:@next/next/recommended
   - prettier
 
-plugins:
-  - prettier
-
-globals:
-  "cy": true
-
 rules:
-  prettier/prettier: warn
-
-  react/react-in-jsx-scope: 0
-  react/jsx-filename-extension: 0
-  react/no-unescaped-entities: 0
-  react/jsx-props-no-spreading: 0
-  react/require-default-props: 0
-  react/no-children-prop: 0
+  react/react-in-jsx-scope: off
+  react/jsx-filename-extension: off
+
   react-hooks/exhaustive-deps: off
 
-  no-alert: 0
-  no-console: 0
-  no-nested-ternary: 0
-  max-classes-per-file: 0
-  class-methods-use-this: 0
+  no-console: off
 
-  import/prefer-default-export: 0
+  class-methods-use-this: off
 
   "@typescript-eslint/no-non-null-assertion": off
-  "@typescript-eslint/no-unused-vars": ["warn", { ignoreRestSiblings: true }]
```

Comment:
This was a good warning, unused variables should start with a _ and otherwise be detected?

Reply:
hmm indeed thanks, I'll check